src/github.js

```diff
@@ -23,15 +25,28 @@ const GitHub = (config) => {
   }
 
   const fetchGitHub = async (path, method = 'GET', body) => {
-    const res = await fetch(
-      `https://api.github.com/repos/${config.owner}/${config.repo}/${path}`,
-      {
-        body: JSON.stringify(body),
-        headers: headers,
-        method: method
-      }
+    const id = uniqueId()
+    const url = (
+      `https://api.github.com/repos/${config.owner}/${config.repo}/${path}`
     )
 
+    if (config.isVerbose) {
+      logHint(`Fetch #${id}: ${JSON.stringify({
+        url: url,
+        method: method,
+        body: body
+      }, null, 2)}`)
+    }
```

Comment:
Minor stylistic suggestion... I prefer the shortcut `{prop}` to the "full" `{prop: prop}` notation.

That would give something like:
```javascript
logHint(`Fetch #${id}: ${JSON.stringify({url, method, body}, null, 2}`)
```